src/ResourceManager/DevTestLabs/Commands.DevTestLabs/Commands/DtlPolicyCmdletBase.cs

```diff
@@ -0,0 +1,60 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.DevTestLabs
+{
+    public class DtlPolicyCmdletBase : DevTestLabsCmdletBase
+    {
+        #region Input Parameter Definitions
+
+        protected const string ParameterSetEnable = "Enable";
+        protected const string ParameterSetDisable = "Disable";
+
+        /// <summary>
+        /// Enable.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            Position = 2,
+            ParameterSetName = ParameterSetEnable,
+            HelpMessage = "Whether to enable the policy.")]
+        [ValidateNotNullOrEmpty]
+        public SwitchParameter Enable
```

Comment:
Is it necessary to have explicit parameters for enable and disable?  Can enable be implied from other parameter sets?  I wonder if this creates confusing parameter set issues, since disable usaually only needs parameters for identification, while Enable requires additional parameters.  As it is, this definition of the parameter set in a base cmdlet prevents combining these parameters with any parameter set other than the default.